src/app/screens/confirmBrc20Transaction/brc20FeesComponent.tsx

```diff
@@ -0,0 +1,96 @@
+import { SupportedCurrency } from '@secretkeylabs/xverse-core';
+import { currencySymbolMap } from '@secretkeylabs/xverse-core/types/currency';
+import BigNumber from 'bignumber.js';
+import { useTranslation } from 'react-i18next';
+import { NumericFormat } from 'react-number-format';
+import styled from 'styled-components';
+
+const Card = styled.div`
+  background: ${(props) => props.theme.colors.background.elevation1};
+  border-radius: ${(props) => props.theme.radius(2)}px;
+  padding: ${(props) => props.theme.spacing(8)}px;
+`;
+
+const CardTitle = styled.div`
+  margin-bottom: ${(props) => props.theme.spacing(8)}px;
+`;
+
+const TextLabel = styled.label`
+  ${(props) => props.theme.body_medium_m}
+  color: ${(props) => props.theme.colors.white[200]};
+`;
+
+const TextValue = styled.p`
+  ${(props) => props.theme.body_medium_m}
+  color: ${(props) => props.theme.colors.white[0]};
+`;
+
+const Rows = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${(props) => props.theme.spacing(8)}px;
+`;
+
+const RowItem = styled.div`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const FiatAmountText = styled.p`
+  ${(props) => props.theme.body_medium_s}
+  color: ${(props) => props.theme.colors.white[400]};
+  display: flex;
+  justify-content: flex-end;
+`;
+
+function Brc20FeesComponent({
```

Comment:
🤔 yes I guess that makes sense if I click X or cancel. will change it

Reply:
I committed a change to editFees which reverts the change on click cancel. it results in a flicker of the estimated fees when the user closes editFees modal, but I think it's OK.

otherwise i'd have to refactor to do more state management in the parent component

Reply:
Cool, that's a better solution 👍 Just a note to test what happens if you set the fee rate to like 20 and then click on cancel. Is the confirm button disabled until the page settles with the new fees or can you cancel the fee change and immediately click on confirm? The former is preferable if possible.

Reply:
can disable the cancel and confirm buttons while fee is loading 👍